models/Document/Dao.php

```diff
@@ -377,11 +377,21 @@ public function getChildAmount($user = null)
     /**
      * Checks if the document has siblings
      *
+     * @param bool $unpublished
+     *
      * @return bool
      */
-    public function hasSiblings()
+    public function hasSiblings($unpublished = false)
```

Comment:
please clarify the meaning of $unpublished. Maybe find a better name for the flag and/or document it in the phpdoc.

Reply:
maybe `$includingUnpublished` ?

Reply:
The same would be also for `getChildren()` and possibly others.

Reply:
done!